src/handbook/community/community-guidelines.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Can we remove duplication between: https://flowfuse.com/handbook/company/communication/#node-red-community-interactions.

I think that section I just linked should be removed and the contents, where not yet done, to be present in these pages.

Reply:
As outreach is a marketing activity, I think this should all be under `/handbook/customer/community/*`.

Reply:
+1 @sumitshinde-84 can you please move it.